.github/scripts/trymerge.py

```diff
@@ -301,6 +301,7 @@
     re.MULTILINE
 )
 RE_REVERT_CMD = re.compile(r"@pytorch(merge|)bot\s+revert\s+this")
+RE_REVERT_CMD_CLI = re.compile(r"@pytorch(merge|)bot\s+revert\s+(-m.*|-c.*)(-m.*|-c.*)")
```

Comment:
Question, why (-m.*|-c.*)(-m.*|-c.*) is included 2 times ?
We are trying to match :
pytorchbot revert -m "Test" -c "test"
pytorchbot revert -c "Test" -c "test"

Reply:
In my mind, it makes sense to do it only on the test-infra side but maybe they wanted to have some additional checks here as well? Like they check if the comments been edited as well which can be a useful check.